apps/desktop/src/renderer/screens/main/components/CommandPalette/CommandPalette.tsx

```
@@ -1,107 +1,158 @@
-					</>
-				);
-			}}
-		/>
+						{results.map((file, index) => (
+							<div
+								key={file.id}
+								data-selected={index === selectedIndex || undefined}
+								className="data-[selected]:bg-accent data-[selected]:text-accent-foreground [&_svg:not([class*='text-'])]:text-muted-foreground relative flex cursor-default items-center gap-2 rounded-sm px-2 py-2 text-sm outline-none select-none [&_svg]:pointer-events-none [&_svg]:shrink-0 [&_svg:not([class*='size-'])]:size-4"
+								onClick={() => handleSelectFile(file.path)}
+								onMouseMove={() => setSelectedIndex(index)}
+							>
```

Biome (2.4.9)

[error] Static Elements should not be interactive.

(lint/a11y/noStaticElementInteractions)